packages/cli/src/commands/init/index.ts

```diff
@@ -20,25 +20,11 @@ export default async function init(cwd: string) {
 
   if (fs.existsSync(changesetBase)) {
     if (!fs.existsSync(path.join(changesetBase, "config.json"))) {
```

Comment:
Question:
Is this block responsible for creating a default template config file for changeset usage ?

If yes, considering we are extending the support for a `.js` config 
1. Shouldn't we allow the user to specify the config (js or json) format during the init ? (could be using a command line flag)
2. The current [read](https://github.com/atlassian-forks/changesets/pull/6/files#diff-2eafc75a9015704b121dff9585728616f562e1625da9a53e44a86cacb94ae619R100) implementation checks for `.js` first and then the json. If the used prefers to use a `js` file this command would fail, correct ?

**Suggestion**
Can we have an implementation where the `js` extends the `json` ? In that case I can understand this usage where the `json` gets created and the `js` overrider and field that might exits. IMO this addresses 2 usecases 
1. If the usage is json it can be extended to js.
2. The usage can also be extended to support a complete js based config without limiting the usage to either.

Reply:
@asakharia 
> Is this block responsible for creating a default template config file for changeset usage?

yes

> The current [read](https://github.com/atlassian-forks/changesets/pull/6/files#diff-2eafc75a9015704b121dff9585728616f562e1625da9a53e44a86cacb94ae619R100) implementation checks for .js first and then the json. If the user prefers to use a js file this command would fail, correct ?

If there is a .js config [read](https://github.com/atlassian-forks/changesets/pull/6/files#diff-2eafc75a9015704b121dff9585728616f562e1625da9a53e44a86cacb94ae619R100) will read the js, otherwise, it will read .json config.
It would fail only if none of the two configs is present, this [try-catch](https://github.com/atlassian-forks/changesets/pull/6/files#diff-df960c2b907455dd827b0ac9f55518091f3632fb6114b83465e909d07efdf97bL43) block was meant only to intercept error that was caused by the old config being present. 
You may clone this branch and see how it works.

Reply:
My apologies. I assumed the error would simply fail the command. 
Now that you mentioned it, the implementation makes sense.